content/about/code-of-conduct.md

```diff
@@ -2,4 +2,105 @@
 title = "Code of Conduct"
 +++
 
-We are in the process of drafting one. Coming soon ...
+
+# Code of conduct
+
+> This code of conduct is copied and adapted from the code of conduct used at the
+> [Collaborations Workshop 2020](https://www.software.ac.uk/cw20/code-conduct)
+> (CC-BY-NC 2.5 license) which was adapted from the example policy at the
+> [Geek Feminism wiki](https://geekfeminism.wikia.org/wiki/Conference_anti-harassment/Policy),
+> created by the Ada Initiative and other volunteers (CC-0 license).
+> The procedure for reporting harassment has been adopted from the Ada Initiative's guide titled
+> ["workshop anti-harassment/Responding to Reports"](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Responding_to_reports).
+
+We value the participation of each stakeholder and want our members and
+participants to our events to have an enjoyable and fulfilling experience.
+Accordingly, all members and participants are expected to show respect and
+courtesy to other members and participants through all communication channels.
+
+To make clear what is expected, all Nordic-RSE members as well as participants,
+speakers, exhibitors, organisers and volunteers at Nordic-RSE events are
+required to conform to the following code of conduct. Organisers will enforce
+this code throughout the event.
+
+
+### Summary
+
+We are dedicated to providing a harassment-free experience for everyone. We do
+not tolerate harassment in any form.
+
+All communication should be appropriate for a professional audience including
+people of many different backgrounds.
+
+Be kind to others. Do not insult or put down others.
+
+Behave professionally. Remember that harassment, unprofessional remarks and
+messages, and exclusionary jokes are not appropriate in Nordic-RSE.
+
+Members violating these rules may have their membership revoked by the board.
+Participants to events violating these rules may be asked to leave without a
+refund at the sole discretion of the conference organisers.
+
+Thank you for helping make this a welcoming, friendly event for all.
```

Comment:
Maybe I would not leave this sentence here _Thank you for helping make this a welcoming, friendly event for all._

Reply:
Missed that one :) Will fix.